src/main/java/uk/gov/ons/ctp/integration/mockcaseapiservice/CasesConfig.java

```diff
@@ -50,7 +46,7 @@ public void setCases(final String cases) throws IOException, CTPException {
     final ObjectMapper objectMapper = new ObjectMapper();
     final List<CaseContainerDTO> caseList =
         objectMapper.readValue(cases, new TypeReference<List<CaseContainerDTO>>() {});
-    addData(caseList);
+    addOrReplaceData(caseList);
```

Comment:
now you have changed the functionality of the "add" endpoint, I thing the endpoint should rename "add" to "save".

Reply:
That's a good idea. Phil and I discussed a few names but couldn't really come up with anything. I will suggest "save" to him.